test/OrchardCore.Tests/Apis/ContentManagement/DeploymentPlans/ContentStepLuceneQueryTests.cs

```diff
@@ -36,23 +37,39 @@ public async Task ShouldUpdateLuceneIndexesOnImport()
 
             // Search indexes are no longer updated in a deferred task at the end of a shell scope
             // but in a background job after the http request, so they are not already up to date.
-            await Task.Delay(2_000);
 
-            // Test
-            var result = await context
-                .GraphQLClient
-                .Content
-                .Query("RecentBlogPosts", builder =>
-                {
-                    builder
-                        .WithField("displayText");
-                });
+            var timeoutTask = Task.Delay(5_000);
+
+            while (true)
+            {
+                await Task.Delay(1_000);
+                // Test
+                var result = await context
+                    .GraphQLClient
+                    .Content
+                    .Query("RecentBlogPosts", builder =>
+                    {
+                        builder
+                            .WithField("displayText");
+                    });
 
-            var nodes = result["data"]["recentBlogPosts"];
+                var nodes = result["data"]["recentBlogPosts"];
 
-            Assert.Equal(2, nodes.AsArray().Count);
-            Assert.Equal("new version", nodes[0]["displayText"].ToString());
-            Assert.Equal("second content item display text", nodes[1]["displayText"].ToString());
+                if (nodes is not null
+                    && nodes.AsArray().Count == 2
+                    && "new version".Equals(nodes[0]["displayText"].ToString())
+                    && "second content item display text".Equals(nodes[1]["displayText"].ToString())
+                    )
+                {
+                    Assert.True(true);
```

Comment:
This is a no-op.

Reply:
Sorry, I don't see what you mean. Do I need to change the code here?

Reply:
updated